examples/components/WithResponsiveProps/withResponsiveProps.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import styled from 'supported-styling-library'
-import { useResponsiveComponent } from 'atomic-layout'
+import { withResponsiveProps } from 'atomic-layout'
```

Comment:
What do you think of naming it `makeResponsive()`?

Reply:
agree,  I think it makes more sense :)